apps/frontend/src/app/Protect.js

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from "react";
```

Comment:
@Nazi-pikachu can we move this in folder name `utils`

Reply:
Resolved